src/components/NcIconSvgWrapper/NcIconSvgWrapper.vue

```diff
@@ -113,6 +183,15 @@ export default {
 	name: 'NcIconSvgWrapper',
 
 	props: {
+		/**
+		 * Make the icon directional, meaning it is langauge direction aware.
+		 * If the icon is placed in a right-to-left context it will be mirrored vertically.
+		 */
+		directional: {
+			type: Boolean,
+			default: false,
+		},
```

Comment:
Now this PR is `feat(NcIconSvgWrapper)` :P

Reply:
feature fix ;)